packages/react/src/components/Panel/PanelContent.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+export interface PanelContentProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  children: ReactNode;
+  className?: string;
+  padding?: boolean;
+}
+
+const PanelContent = ({
```

Comment:
This should include a ref (via `React.forwardRef`) according to our [component standards](https://cauldron.dequelabs.com/component-guidelines#component-standards).